### ISSUE TITLE
Feature: Port forwarding

### ISSUE THREAD

**Issue by blacktea** [typicode/hotel#149]
How could we implement port forwarding?

In case I have multiple servers on 4001, 4002, 4003... and I would like to have them in project.dev:80, api.project.dev:80, api2.project.dev:80?

At the moment I'm using local Apache server with mod_proxy_(http)
How could I achieve this with Hotel?

**Comment by typicode** [comment]
Hi @blacktea,

You can pass URLs instead of commands:

```
hotel add --name project
hotel add http://127.0.0.1:4002 --name api.project
hotel add http://127.0.0.1:4003 --name api2.project
```

It should forward `project.dev` to ` http://127.0.0.1:4001`

**Comment by typicode** [comment]
You can also have your servers started by hotel on some specific ports too:

```sh
# if your server takes no port option
# or listens on PORT environment variable
hotel add 'command1' --port 4001

# if your server takes a port option
# and you want it to be set to 4002
hotel add 'command2 -p $PORT' --port 4002
```

**Comment by blacktea** [comment]
Yes, but in the end I want all of them to be accessible from port 80.
And I cannot start 3 servers on port 80

**Comment by typicode** [comment]
To access them on port 80, you need to configure your browser or system proxy to use hotel's proxy.pac:
https://github.com/typicode/hotel/blob/master/docs/README.md

After that, you'll be able to access hotel.dev:80, project.dev:80 or simply hotel.dev, project.dev, ...

**Comment by blacktea** [comment]
Yep I did this port configuration the first day, but chrome just kept redirecting me to localhost with specific port

**Comment by typicode** [comment]
Strange, what's your OS?

**Comment by blacktea** [comment]
win 10 64 bit

**Comment by typicode** [comment]
thanks, will try to reproduce :)

**Comment by LeoColomb** [comment]
> but chrome just kept redirecting me to localhost with specific port

I was facing the same problem (exact same system, browser), and it's just Chrome HTTP answers caching. Clearing the cache solves the issue.
